### PR TITLE
Update serving.md.

### PR DESCRIPTION
**Pull request by justinxzhao**

### CODE REVIEW

File: docs/user_guide/serving.md [hunk 1]
```diff
@@ -20,31 +23,29 @@ The structure of the input to the REST API and the structure of the output that
...
 ```
 
 ### Output format
 
-The output format is  a JSON that is independent of the number of inputs and their data types, it only depends on the number of outputs the model pipeline was trained to predict and their data types.
-
-At the moment, Ludwig can predict binary, numerical, categorical, set, sequence and text outputs.
+The output format is a JSON that depends on the number and types of outputs the model pipeline was trained to predict.
 
 For binary outputs, the JSON structure returned by the REST PI is the following:
```
Comment (dantreiman) [inline docs/user_guide/serving.md:46]: s/REST PI/REST API    (or REST Endpoint)  .  Find-replace in the file, this typo is copy/pasted repeatedly.

File: docs/user_guide/serving.md [hunk 2]
```diff
@@ -20,31 +23,29 @@ The structure of the input to the REST API and the structure of the output that
...
+```bash
 curl http://0.0.0.0:8000/predict -X POST -F 'text=mixed together with' -F 'image=@path_to_image/example.png'
 ```
 
 ### Output format
 
-The output format is  a JSON that is independent of the number of inputs and their data types, it only depends on the number of outputs the model pipeline was trained to predict and their data types.
-
-At the moment, Ludwig can predict binary, numerical, categorical, set, sequence and text outputs.
+The output format is a JSON that depends on the number and types of outputs the model pipeline was trained to predict.
```
Comment (dantreiman) [inline docs/user_guide/serving.md:44]: Suggestion:  The response is a JSON dictionary with keys prefixed by the names of the model's output features.

**Review by dantreiman — CHANGES_REQUESTED** [review]

File: docs/user_guide/serving.md (hunk 1 above)
Comment (justinxzhao) [reply docs/user_guide/serving.md]: Done.

File: docs/user_guide/serving.md (hunk 2 above)
Comment (justinxzhao) [reply docs/user_guide/serving.md:44]: Done.

**Pull request merged**